### ISSUE TITLE
UTF-8 characters break PascalCase checking

### ISSUE THREAD

**Issue by mariszin** [vuejs/eslint-plugin-vue#688]
When using UTF8 characters in component name, the plugin fails to recognize PascalCase name.

* **ESLint version:** v4.19.1	
* **eslint-plugin-vue version:** v4.7.1
* **Node version:**	v8.11.3

**Please show your full configuration:**
```js
module.exports = {
  root: true,
  env: {
    node: true
  },
  'extends': [
    'plugin:vue/recommended',
    '@vue/airbnb'
  ],
  rules: {
    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off'
  },
  parserOptions: {
    parser: 'babel-eslint'
  }
};
```

**What did you do?**	
```vue
export default {
  name: 'DarbībaShow',
};
```

**What did you expect to happen?**
linter should recognize this as a valid PascalCase

**What actually happened?**
```
...\show.vue
  13:9  error  Property name "DarbībaShow" is not PascalCase  vue/name-property-casing

✖ 1 problem (1 error, 0 warnings)
  1 error, 0 warnings potentially fixable with the `--fix` option.
```
